core/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     compile "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
     compile 'com.google.googlejavaformat:google-java-format:1.10.0'
     compile "org.opensearch:common-utils:${common_utils_version}"
+    compile 'commons-validator:commons-validator:1.7'
```

Comment:
This will be moved to alerting if we move the LocalUriInput class to alerting from core